### DISCUSSION TITLE
Spacing for UniformItemsLayout

### DISCUSSION THREAD

**Discussion by varyamereon** [CommunityToolkit/Maui#436] [New Feature Discussions]
The UniformItemsLayout performs a similar function to Grid and StackLayout, and therefore should include spacing as this is a standard property for these layouts that saves adding margins to every item within the layout.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
